shiny/ui/_input_select.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import annotations
 
+from shiny.types import Jsonifiable
```

Comment:
```suggestion
from ..types import Jsonifiable
```

Reply:
Oops, that was a vscode auto-import. I think I have the released shiny installed and not the editable local package.